src/super_gradients/recipes/imagenet_mobilenetv3_base.yaml

```diff
@@ -2,6 +2,7 @@
 
 defaults:
   - training_hyperparams: imagenet_mobilenetv3_train_params
+  - dataset_params: imagenet_dataset_params
```

Comment:
Don't think this file was supposed to be in this PR...

Reply:
This was part of previous PR that is now merged. Now it is removed fro this PR.